<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="carte">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="TauxSnirium">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Taux snirium :</string>
    </property>
   </widget>
   <widget class="QLabel" name="DistanceObstacle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Distance obstacle :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepTauxSnirium">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepPosition">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepIP">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>192.168.1.171</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="Batterie">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>910</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepAngle">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>860</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepDistanceObstacle">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>810</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Angle">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>860</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Angle :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Position">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>810</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Distance :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeConnecter">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>40</y>
      <width>150</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 163, 2);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>910</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Batterie :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ip">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Adresse IP :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeDeconnecter">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 0, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>DÃ©connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RepPort">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1664</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="MonterBras">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â§</string>
    </property>
    <property name="shortcut">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BaisserBras">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>290</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â©</string>
    </property>
    <property name="shortcut">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="St">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 50pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>ð</string>
    </property>
    <property name="shortcut">
     <string>T</string>
    </property>
   </widget>
   <widget class="QLabel" name="fond">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1900</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ga">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 50pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â¦</string>
    </property>
    <property name="shortcut">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ar">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>570</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 50pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â©</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rec_trame">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 6pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>rÃ©cupÃ©rtion de tram</string>
    </property>
    <property name="shortcut">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dr">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>400</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 50pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â¨</string>
    </property>
    <property name="shortcut">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Av">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>230</y>
      <width>170</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>50</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 50pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>â§</string>
    </property>
    <property name="shortcut">
     <string>Z</string>
    </property>
   </widget>
   <zorder>fond</zorder>
   <zorder>TauxSnirium</zorder>
   <zorder>DistanceObstacle</zorder>
   <zorder>RepTauxSnirium</zorder>
   <zorder>RepPosition</zorder>
   <zorder>RepIP</zorder>
   <zorder>Batterie</zorder>
   <zorder>RepAngle</zorder>
   <zorder>Port</zorder>
   <zorder>RepDistanceObstacle</zorder>
   <zorder>Angle</zorder>
   <zorder>Position</zorder>
   <zorder>SeConnecter</zorder>
   <zorder>label</zorder>
   <zorder>Ip</zorder>
   <zorder>SeDeconnecter</zorder>
   <zorder>RepPort</zorder>
   <zorder>carte</zorder>
   <zorder>MonterBras</zorder>
   <zorder>BaisserBras</zorder>
   <zorder>St</zorder>
   <zorder>Ga</zorder>
   <zorder>Ar</zorder>
   <zorder>rec_trame</zorder>
   <zorder>Dr</zorder>
   <zorder>Av</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1429</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
